<ui version="4.0" >
 <class>MyHeadset</class>
 <widget class="QWidget" name="MyHeadset" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Device Info</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="headsetTabWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="headsetSheet" >
      <attribute name="title" >
       <string>Headset</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>2</number>
       </property>
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="profileLabel" >
           <property name="text" >
            <string>Headset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="headsetInfo" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text" >
            <string>00:00:00:00:00:00</string>
           </property>
           <property name="maxLength" >
            <number>17</number>
           </property>
           <property name="cursorPosition" >
            <number>17</number>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="connectButton" >
           <property name="text" >
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="volumeSheet" >
      <attribute name="title" >
       <string>Audio</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>2</number>
       </property>
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Volume</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>2</number>
          </property>
          <property name="spacing" >
           <number>1</number>
          </property>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="spkrVol" >
            <property name="maximum" >
             <number>15</number>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="0" column="1" >
           <widget class="QSpinBox" name="micVol" >
            <property name="maximum" >
             <number>15</number>
            </property>
           </widget>
          </item>
          <item rowspan="3" row="0" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Microphone:</string>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Speaker:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Audio Input/Output</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="audioIO" >
            <item>
             <property name="text" >
              <string>Handset</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Headset</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
